src/address/map-to-internal-address.ts

```diff
@@ -10,7 +10,7 @@ import InternalAddress from './internal-address';
  * the transition period as we are moving to adopt the new storefront API object
  * schema.
  */
-export default function mapToInternalAddress(address: Address | BillingAddress, consignments?: Consignment[]): InternalAddress {
+export default function mapToInternalAddress(address: Address | BillingAddress, consignments?: Consignment[]): InternalAddress<any> {
```

Comment:
Maybe it's better to say `InternalAddress<string | number>`?

Reply:
If I change it to `string | number` it causes like 50 errors, might be good to keep it as `any` ?

I tried setting `<T extends string | number>`  but linter was still complaining